src/cdk-experimental/menu/menu-group.spec.ts

```diff
@@ -46,8 +46,8 @@ describe('MenuGroup', () => {
       fixture.detectChanges();
 
       menuItems = fixture.debugElement
-        .queryAll(By.directive(CdkMenuItem))
-        .map(e => e.injector.get(CdkMenuItem));
+        .queryAll(By.directive(CdkMenuItemRadio))
+        .map(e => e.injector.get(CdkMenuItemRadio));
```

Comment:
Here and elsewhere, the test code may be a little clearer if you replace 'e' with a more descriptive name (such as debugElement or menuItemRadioDebugElement). If you're following the CDK convention, however, you should probably continue doing so.

Reply:
ya, [some can be quite long](https://github.com/angular/components/blob/master/src/material/slider/slider.ts#L11-L57)